extra/modules/ortb2-blocking/src/main/java/org/prebid/server/hooks/modules/ortb2/blocking/core/model/Result.java

```diff
@@ -0,0 +1,28 @@
+package org.prebid.server.hooks.modules.ortb2.blocking.core.model;
+
+import lombok.Value;
+
+import java.util.List;
+
+@Value(staticConstructor = "of")
+public class Result<T> {
+
+    private static final Result<?> EMPTY = of(null, null);
+
```

Comment:
I know I already brought this up once, but would `Option` really not be a safer and more robust alternative?

Either you check everything for `null` making the code harder to follow or we fix NPE along the way they occure.

Option forces me to handle nullable variables and lets me use others without fearing NPE 🤗